py/bitbox02/bitbox02/communication/bitbox_api_protocol.py

```diff
@@ -680,11 +680,13 @@ def reboot(
         return True
 
     @staticmethod
-    def get_info(transport: TransportLayer) -> Tuple[str, Platform, Union[BitBox02Edition], bool]:
+    def get_info(
+            transport: TransportLayer
+    ) -> Tuple[str, Platform, Union[BitBox02Edition], bool, bool]:
         """
-        Returns (version, platform, edition, unlocked).
-        This is useful to get the version of the firmware when a usb descriptor is not available
-        (via BitBoxBridge, etc.).
+        Returns (version, platform, edition, unlocked, initialized).
+        This is useful to get the version of the firmware or the device unlocked/initialized status
```

Comment:
I think the _This is useful (...)_ part could be improved but I don't understand it well enough and also I'm bad at constructing concise sentences.

-> this is useful if there was no handshake yet too, otherwise we could just call the other info request that already has the initialized attribute in the response...how could this be phrased ? (when I tried to use the other request in `api-go` it would throw an error like this: `handshake must come first`

but I guess it is fine to leave it as is?